<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAutoDJ</class>
 <widget class="QWidget" name="DlgAutoDJ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto DJ</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonShuffle">
       <property name="toolTip">
        <string>Shuffle the content of the Auto DJ playlist.</string>
       </property>
       <property name="text">
        <string>Shuffle</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAddRandom">
       <property name="toolTip">
        <string>Add a random track from track sources (crates) or Library to the Auto DJ playlist.</string>
       </property>
       <property name="text">
        <string>Add Random</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSkipNext">
       <property name="toolTip">
        <string>Skip the next track in the Auto DJ playlist.</string>
       </property>
       <property name="text">
        <string>Skip Track</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFadeNow">
       <property name="toolTip">
        <string>Trigger the transition to the next track.</string>
       </property>
       <property name="text">
        <string>Fade Now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxTransition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Determines the duration of the transition.</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>-9</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string comment="&quot;sec.&quot; as in seconds">Seconds</string>
       </property>
       <property name="text">
        <string>sec.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSelectionInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAutoDJ">
       <property name="toolTip">
        <string>Turn Auto DJ on or off.</string>
       </property>
       <property name="text">
        <string>Enable Auto DJ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="m_pTrackTablePlaceholder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/mixxx.qrc"/>
 </resources>
 <connections/>
</ui>
